src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/TripleDesImplementation.cs

```diff
@@ -69,7 +69,7 @@ private ICryptoTransform CreateTransform(byte[] rgbKey, byte[] rgbIV, bool encry
                 // Some platforms do not support Two-Key Triple DES, so manually support it here.
                 // Two-Key Triple DES contains two 8-byte keys {K1}{K2} with {K1} appended to make {K1}{K2}{K1}.
                 byte[] newkey = new byte[24];
-                Array.Copy(rgbKey, 0, newkey, 0, 16);
+                Array.Copy(rgbKey, newkey, 16);
```

Comment:
IMO the original code reads a bit cleaner since it's more obvious that the destination array is being written piecemeal. But honestly it's only a weakly held opinion.